ui/tests/acceptance/saml-auth-method-test.js

```diff
@@ -145,6 +145,26 @@ module('Acceptance | enterprise saml auth method', function (hooks) {
     await click('[data-test-auth-submit]');
   });
 
+  test('it should render API error when it comes from data block', async function (assert) {
```

Comment:
This use case is what prompted the fix, but if we backport this to earlier versions perhaps we add this test in a different PR (or just remove from the backport?)

Reply:
I think it makes sense to add it to a different PR :D